main.rb

```diff
@@ -34,7 +35,10 @@ def main
   loop do
     options_menu
     choice = gets.chomp.to_i
-    exit if choice == 7
+    if choice == 7
+      app.save_people_data
+      exit
+    end
     handle_menu_choice(choice, app)
```

Comment:
- [ ] While exiting the app I got the error shown below please fix this error and you are good to go. `the problem is app.save_people_data is undefined` please make sure that `On exit from the app the data should be saved (all the data.). For now, data is properly saving but make sure after fixing the issue is works as now.
### Error
![Screenshot 2023-09-07 145257](https://github.com/HFG43/OOP-school-library/assets/81727988/dced52c8-f7ab-4a57-a3e5-795d2a113621)